app/models/talk.rb

```diff
@@ -333,4 +338,20 @@ def suggestion_summary
       at #{event.name}
     HEREDOC
   end
+
+  def set_kind
+    self.kind =
+      case title
+      when /.*(keynote:|opening\ keynote|closing\ keynote).*/i
+        :keynote
+      when /.*workshop:.*/i
+        :workshop
+      when /.*panel:.*/i
+        :panel
+      when /.*lightning\ talk: .*/i
+        :lightning_talk
+      else
+        :standard
```

Comment:
I'm wondering if we should use `:talk` instead of `:standard` so we could humanize the `kind` column when we are showing it the UI.

Reply:
I like it!